docs(managed-databases): add encryption key size (#6661)
scaleway/docs-content: pages/managed-databases-for-postgresql-and-mysql/concepts.mdx

Once activated on a Database Instance, encryption at rest cannot be disabled.
</Message>

> Encryption at rest allows you to permanently encrypt your database data. The data is encrypted at volume level using [LUKS](https://en.wikipedia.org/wiki/Linux_Unified_Key_Setup). At Scaleway, `aes-xts-plain64` with `sha256` and a 512-bit key size (256 bits × 2 keys for XTS mode) is used for all Database Instances. The management of the encryption key is done by Scaleway.

Reviewer: ```suggestion
[Encryption at rest](/managed-databases-for-postgresql-and-mysql/concepts/#encryption-at-rest) lets you permanently encrypt your database data at the volume level using [LUKS](https://en.wikipedia.org/wiki/Linux_Unified_Key_Setup). All Database Instances at Scaleway use `aes-xts-plain64` with `sha256` and a 512-bit key size (256 bits × 2 keys for XTS mode). Scaleway manages the encryption key on your behalf.
```
